core/src/apps/cardano/README.md

```diff
@@ -0,0 +1,205 @@
+# Cardano
+
+MAINTAINER = ??
+
+ORIGINAL AUTHOR = Juraj Muravský <juraj.muravsky@vacuumlabs.com>
+
+SHELLEY UPDATE AUTHOR = Gabriel Kerekeš <gabriel.kerekes@vacuumlabs.com>
+
+REVIEWER = Tomas Susanka <tomas.susanka@satoshilabs.com>
```

Comment:
Please add `Jan Matejek <jan.matejek@satoshilabs.com>` as a reviewer as well, ideally on a first position. He was the main reviewer.